### ISSUE TITLE
Set aria-expanded attribute true/false to match drawer visibility

### ISSUE THREAD

**Issue by kirsty-hames** [adaptlearning/adapt-contrib-pageLevelProgress#188]
### Subject of the issue
As per https://github.com/adaptlearning/adapt-contrib-core/issues/409, buttons and links that unfold / collapse content on the page are to be coded with aria-expanded attribute (according to _success criteria 4.1.2 Name, Role, Value_).

### Expected behaviour
`.nav__pagelevelprogress-btn` should have the relevant `aria-expanded` attribute applied to match drawer visibility.

### Actual behaviour
No such attribute exists.
